config/locales/en.yml

```diff
@@ -196,6 +196,9 @@ en:
       edition_with_appointment:
         one: Edition with appointment
         other: Editions with appointment
+      editionable_worldwide_organisation:
```

Comment:
Can you run the rake task so these are in all the locales?

Reply:
Done.  Involved quite a bit of hacking as the rake task doesn't correctly populate the plural rules 😬.

Reply:
I'm assuming that's why some are `one, other` and others have a load more numbers?